lib/seahorn/BvOpSem2ExtraWideMemMgr.cc

```diff
@@ -544,6 +547,11 @@ class ExtraWideMemManager : public OpSemMemManagerBase {
 
   Expr isDereferenceable(PtrTy p, Expr byteSz) {
     // size should be >= byteSz + offset
+    if (m_ctx.shouldSimplify()) {
+      ScopedStats _st_("opsem.simplify");
+      p = PtrTy(m_ctx.getSimplifier()->simplify(p.toExpr()));
```

Comment:
does this simplification has a big effect? 

I'd imagine bigger effect on size and offset portion

Reply:
Good point! I fixed to only simplify offset, size sampling we don't seem to be spending much time in `simplify` but rather `solver::check_sat()`